lib/browser/base.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, window_size_height, windows_size_width, **kwargs):
         self.windows_size_width = str(windows_size_width)
         self.get_browser_settings(**kwargs)
         self.test_env = os.environ.copy()
+        if self.current_platform_name == "linux2":
+            self.test_env['LANGUAGE'] = "en-US"
```

Comment:
how about putting these two line in `BrowserChrome. get_browser_settings ()` method?